### DISCUSSION TITLE
Hi :) I am Gonzalo, this course is a great idea, thank you so much for all the effort.

### DISCUSSION THREAD

**Discussion by gonzalo-munillag** [OpenMined/courses#117] [Introduce Yourself]
Hi :)
Following Yemi's instructions, I am introducing myself haha

I am Gonzalo, I am doing my Ph.D. in privacy preservation at TU Munich. It has been a year so far.
My motivation for joining and completing this set of courses is to learn a lot more about PETs code-wise, the underlying mathematical foundations of the different PETs, and their use cases.

Privacy is a tough sale, looking forward to making it easier.

I also have a blog about privacy and DP: https://github.com/gonzalo-munillag/Blog; in case you are interested.

Cheers!

**Comment by yemikifouly** [comment]
Hello Gonzalo! 👋 Welcome to OpenMined! Privacy can indeed be a tough sale but I think that the courses will help make it easier. 🙂 Did you start the course yet? The first course, the one currently available, is not technical. Rather, it discusses issues around privacy, invites us to think more deeply about the ways we might define it and then discusses technical concepts that can help build solutions. The courses that will be released after this one will be a lot more technical and you will get to learn more about the principles underlying the different PETs and their use cases.

I hope you enjoy the first course and stick around for the following ones. Once again, welcome to OpenMined! 🙂

**Reply by gonzalo-munillag** [reply]
Woh Yemi, you are all in. I did not expect such a kind reply. Awesome people in OpenMined, keep it up!

**Reply by yemikifouly** [reply]
Thanks Gonzalo! 🙂 You will find that everyone at OpenMined is kind and very helpful. It's the reason why I, myself, enjoy being part of this community. 😀
